changelog/unreleased/example-deployments.md

```diff
@@ -0,0 +1,6 @@
+Change: disable the color logging in docker compose examples
+
+Disabled the color logging in the example docker compose deployments.
```

Comment:
Could you add easy to understand reasoning for preferring monochrome log output? :-) from a dev perspective this was not entirely clear to me until I got your explanation.

Reply:
Something like `Although being helpful from a developer point of view, color logs are considered counterproductive for deployments as they are xyz (slow, harder to filter, ...)`